### PR TITLE
Set the correct Angle GLSL output  when using WebGL 2

### PR DESCRIPTION
**Pull request by MortimerGoro**
Set the correct Angle GLSL output  when using WebGL 2

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [x] There are tests for these changes OR
- [ ] These changes do not require tests because _____

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/19192)

### CODE REVIEW

File: components/script/dom/webglshader.rs [hunk 1]
```diff
@@ -40,11 +41,20 @@ pub struct WebGLShader {
     renderer: WebGLMsgSender,
 }
 
-#[cfg(not(target_os = "android"))]
-const SHADER_OUTPUT_FORMAT: Output = Output::Glsl;
+#[cfg(not(any(target_os = "android", target_os = "ios")))]
+const SHADER_OUTPUT_FORMAT_WEBGL1: Output = Output::Glsl;
+// MacOS Requires Glsl150Core because it uses a OpenGL 3.2 Core Context.
```
Comment (anholt) [inline components/script/dom/webglshader.rs]: Instead of a bunch of per-OS hacks here, we should probably be looking at SHADING_LANGUAGE_VERSION

Comment (MortimerGoro) [reply components/script/dom/webglshader.rs]: Great idea! I'll update the PR to use SHADING_LANGUAGE_VERSION

**Comment by MortimerGoro** [comment]
Updated the PR to use SHADING_LANGUAGE_VERSION

**Pull request merged**